samtranslator/translator/translator.py

```diff
@@ -111,7 +111,7 @@ def translate(
         self.feature_toggle = (
             feature_toggle
             if feature_toggle
-            else FeatureToggle(FeatureToggleDefaultConfigProvider(), stage=None, account_id=None, region=None)  # type: ignore[no-untyped-call, no-untyped-call]
+            else FeatureToggle(FeatureToggleDefaultConfigProvider(), stage=None, account_id=None, region=None)
```

Comment:
Does this actually fix the problem?  Since we construct `FeatureToggleAppConfigConfigProvider` every time `translate` is invoked, and `FeatureToggleAppConfigConfigProvider.__init__` invokes `app_config_data_client.start_configuration_session`, it seems like we'd still be retrieving the config on every request.  Or am I misreading this?

Reply:
move discussion to offline